### PR TITLE
add GitHub scavenger hunt

### PR DESCRIPTION
**Pull request by jsms90**
For the 1st business development session, get students navigating around the FAC repos with a "scavenger hunt" - `international`, `<city>-programme` (and committees from there) & `master-reference` itself.

What we're getting students to find in `master-reference`:
+ vision doc
+ who the PR reviewers are
+ available gitter channels
+ coursebook folder & general week structure / week 1 structure

Finally, get students to raise an issue!!
+ correct issue-raising process
+ get feedback on precourse material #93 

Relates #334

### CODE REVIEW

**Comment by eliascodes** [comment]
@jsms90 it's not clear to me how this relates to #273, they look quite similar.

File: coursebook/general/github-scavenger-hunt.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+## London's GitHub Scavenger Hunt
+
+From our curriculum to the running of our campuses, at Founders and Coders we track all our discussions and decision making via GitHub. By using GitHub as our single source of truth, we reduce the cognitive overload that's involved when constantly switching between mediums of communication.
+
+We aim for hyper-transparency in the running of the organisation, to enable the greatest level of contribution from our members. In order to get involved, you will need to get acquainted with our GitHub repos.
```
Comment (des-des) [inline coursebook/general/github-scavenger-hunt.md]: I think `transparency` is fine

File: coursebook/general/github-scavenger-hunt.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#### Overview
++ What is the master reference aiming to do?
++ Who are the maintainers of the master reference i.e. who is available to review pull requests?
++ Which gitter channel should you post in if you wanted to share a cool website or blogpost that you've found?
+
+#### Schedule
+The coursebook contains some general overview files, as well as the content of each week.
++ What will you be doing after reading week?
++ How many hours do you spend on business development / community engagement per week?
++ When do you projects start and finish?
```
Comment (des-des) [inline coursebook/general/github-scavenger-hunt.md]: `you` -> `your`

File: coursebook/general/github-scavenger-hunt.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
++ How long do you have to prepare your presentations?
++ Where can you find extra resources that might compliment your learning during the week?
+
+#### How to contribute
+Our curriculum undergoes continuous iterative improvement. The master reference is an open source project and we greatly appreciate all contributions. Here is your first chance to share some of your feedback.
+1. Think back to the precourse material you have recently completed. Did you experience any problems? Is there anything you would like to notify us about? Would you make any changes?
```
Comment (des-des) [inline coursebook/general/github-scavenger-hunt.md]: :heart:

**Pull request merged**